infra/main.bicep

```diff
@@ -284,6 +286,36 @@ module openAiSecondary './modules/ai/cognitiveservices.bicep' = if (secondaryOpe
   }
 }
 
```

Comment:
Perhaps add a comment here describing what this does? (or link to the README).

Reply:
changed location to appconfig deployment and added some comments